source/include/TypeAndDataManager.h

```diff
@@ -933,19 +933,13 @@ typedef UInt32 RefNum;
 
 class RefNumVal {
```

Comment:
It still differentiates the uses of refnum's as allocated by the TypeManager for a VIA file versus simple refnums used internally, so I'd rather keep it just for that, and it makes it easy to add fields if we need to do in the future, or inherit from it to do so for a specialized refnum (I almost did this for Queue Refnums instead of adding STAD args to the prims, but decided against it.)